mlflow/environment_variables.py

```diff
@@ -164,3 +164,10 @@ def get(self):
 MLFLOW_SQLALCHEMYSTORE_POOLCLASS = _EnvironmentVariable(
     "MLFLOW_SQLALCHEMYSTORE_POOLCLASS", str, None
 )
+
+#: Specifies the ``timeout_seconds`` parameter to use for ``_run_command`` in the dependency
+#: inference subprocess task.
```

Comment:
Can we make this a bit friendlier for users? I don't think users understand these details, including the subprocess

Reply:
good point! changed!